cmd/mimir/config-descriptor.json

```
@@ -1887,6 +1887,17 @@
           "fieldType": "string",
           "fieldCategory": "experimental"
         },
+        {
+          "kind": "field",
+          "name": "enable_promql_engine_fallback",
+          "required": false,
+          "desc": "If true, fall back to Prometheus' PromQL engine if the streaming engine is in use and it does not support a query.",
```

I tried to reword to make it more clear that it's a per-query fallback and not that a single unsupported query might make all subsequent queries use the Prometheus PromQL engine.

I also set the options argument to be inline code for the Markdown rendering.

The suggestion also avoids using the product name with a possessive which is recently identified area of our documentation style guide: https://developers.google.com/style/possessives#product,-feature,-and-company-names. I'm not certain we need to avoid it here but I do think Prometheus is a trademark so I thought better safe than sorry :) 

```suggestion
          "desc": "If set to `true` and the streaming engine is in use, fall back to using the Prometheus PromQL engine for a queries not supported by the streaming engine.",
```

---

Thanks for the suggestion. I've changed this in d856405. 

I didn't use the inline code formatting for "true" as the option description is never rendered as Markdown anywhere.